diff --git a/dist/classes/directoryHelper.js b/dist/classes/directoryHelper.js
index e0bb07c45e00f0af08a2bc89ff4df406b04e3d0f..1e3dbf829c95902f224706003d6948fe4fe4655e 100644
--- a/dist/classes/directoryHelper.js
+++ b/dist/classes/directoryHelper.js
@@ -22,7 +22,7 @@ class DirectoryHelper {
         if (!this.pathOrDirExists(path))
             return;
         try {
-            fs_1.default.rmdirSync(path, { recursive: true });
+            fs_1.default.rmSync(path, { recursive: true, force: true });
         }
         catch (err) {
             if (err instanceof Error)
